snappy/common_test.go

```diff
@@ -121,19 +133,37 @@ func storeMinimalRemoteManifest(qn, name, origin, version, desc, channel string)
 	return nil
 }
 
-func addDefaultApparmorJSON(tempdir, apparmorJSONPath string) error {
-	appArmorDir := filepath.Join(tempdir, "var", "lib", "apparmor", "clicks")
+func addDefaultApparmorProfile(appid string) error {
+	appArmorDir := dirs.SnapAppArmorDir
+
 	if err := os.MkdirAll(appArmorDir, 0775); err != nil {
 		return err
 	}
 
-	const securityJSON = `{
-  "policy_vendor": "ubuntu-core"
-  "policy_version": 15.04
+	const securityProfile = `
+#include <tunables/global
```

Comment:
This profile is just used in the tests, I renamed the function to make it clearer, its now called addMockDefaultApparmorProfile()  - is that better? If I misunderstood, please let me know.